lib/sda/node.go

```diff
@@ -135,6 +181,19 @@ func (n *Node) EntityList() *EntityList {
 	return n.Tree().EntityList
 }
 
+// List returns the list of TreeNodes cached in the node (creating it if necessary)
+func (n *Node) List() []*TreeNode {
+	if n.treeNodeList == nil {
+		n.treeNodeList = n.Tree().List()
+	}
```

Comment:
Don't we need a mutex here ? It might be called from different go structures. At least I think...